### PR TITLE
Created Login screen

### PR DESCRIPTION
**Pull request by ShrutiJ-01**
Please review and pull this PR
appLoginScreen

### CODE REVIEW

File: lib/login_screen.dart [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+          minimum: EdgeInsets.all(20.0),
+          child: Column(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: [
+              SizedBox(height:35.0),
+              RichText(text: TextSpan(
+                  style: TextStyle(fontSize: 58,color: black_heading,fontWeight: FontWeight.w900,letterSpacing: 2.0),
+                  children:[
+                    TextSpan(text:'Hello \nThere '),
+                    TextSpan(text:'.',style:TextStyle(color: secondary_color,fontWeight: FontWeight.w900)),
```
Comment (pratyushmp) [inline lib/login_screen.dart:34]: Use the Poppins Font family given in the assets folder

File: lib/login_screen.dart [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+              TextField(
+                controller: _emailController,
+                textInputAction: TextInputAction.next,
+                decoration: InputDecoration(
+                  filled: true,
+                  fillColor: textField_fill_color,
+                  contentPadding: EdgeInsets.only(left: 10.0),
+                  enabledBorder: OutlineInputBorder(borderSide:BorderSide(color: textField_fill_color),borderRadius: BorderRadius.circular(12.0)),
+                  focusedBorder: OutlineInputBorder(borderSide:BorderSide(color: textField_fill_color),borderRadius: BorderRadius.circular(12.0)),
+                  hintText: "Type Something..",
```
Comment (pratyushmp) [inline lib/login_screen.dart:52]: Change the hint text to "Enter Email"

File: lib/login_screen.dart [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+                child: Container(
+                  child: Row(
+                    mainAxisSize: MainAxisSize.min,
+                    children: [
+                      TextButton.icon(
+                          onPressed:(){},
+                          icon:Image(
+                              height: 30.0,
+                              width: 30.0,
+                              image: NetworkImage("https://upload.wikimedia.org/wikipedia/commons/thumb/5/53/Google_%22G%22_Logo.svg/768px-Google_%22G%22_Logo.svg.png")),
```
Comment (pratyushmp) [inline lib/login_screen.dart:93]: Use font_awesome_flutter: 8.12.0 for the icons.

File: lib/login_screen.dart [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+              ),
+              SizedBox(height: 15.0),
+              Center(
+                child: RichText(
+                  text: TextSpan(
+                      style:TextStyle(fontSize:18.0,color:secondary_color,fontWeight: FontWeight.w800),
+                      children: [
+                        TextSpan(text:"Don't have an account? "),
+                        TextSpan(
+                            text: "Create one!",
```
Comment (pratyushmp) [inline lib/login_screen.dart:117]: Map "Create One" to the Sign-up Page.

**Review by pratyushmp — CHANGES_REQUESTED** [review]

File: lib/login_screen.dart (hunk 1 above)
Comment (ShrutiJ-01) [reply lib/login_screen.dart:34]: Is it okay if I set it using ThemeData widget in main.dart? that way it will be applied to the entire app

Comment (pratyushmp) [reply lib/login_screen.dart:34]: Yup you can go ahead!

**Review by pratyushmp — APPROVED** [review]

**Pull request merged**